lib/widget_extensions/expanded_extension.dart

```diff
@@ -0,0 +1,8 @@
+part of '../awesome_extensions.dart';
+
+extension ExpandedExtension on Widget {
+  Expanded expanded({int flex = 1}) => Expanded(
+        child: this,
+        flex: flex,
```

Comment:
[Default value for flex param](https://api.flutter.dev/flutter/widgets/Expanded/Expanded.html) is 1.